### DISCUSSION TITLE
SUM Rollups not completing?

### DISCUSSION THREAD

**Discussion by melissajhansen** [jamessimone/apex-rollup#235] [Q&A]
I have a series of rollups configured as base invokable actions in flow.  The are all rolling up data from the Opportunity Contact Role to the Contact record, and are using the same WHERE Clause.  The rollups that use FIRST or LAST work fine, but when I use SUM, the rollup doesn't update the Contact record.  No errors are thrown, but no update is made.  I tried setting the "Full Recalculation Default Number Value" to 0, but that didn't have any effect.

I assume I'm just missing something configuration wise? I just can't figure out what.  It's running as deferred with a commit deferred rollups action.  I'm attaching logs for the synchronous and asynchronous processing (with just the SUM action in use).  

[apex-07L8B0000019LQhUAM.log](https://github.com/jamessimone/apex-rollup/files/7884425/apex-07L8B0000019LQhUAM.log)
[apex-07L8B0000019LUIUA2.log](https://github.com/jamessimone/apex-rollup/files/7884423/apex-07L8B0000019LUIUA2.log)

If anyone has insight, I'd appreciate it!

**Comment by melissajhansen** [comment]
Interestingly, if i use a CMDT-driven Action, configured in the same way as the base action, it works.

**Reply by jamessimone** [reply]
Hmm. Let me take a look through the logs you’ve attached (can’t tell you how much I appreciate you uploading those). This is the first time I’m hearing of a deviation in behavior between the base action and the CMDT-driven one, at least in the way you’re describing! I may ask some follow-up questions tomorrow, as time allows; that’s also when I hope to release the next version.

**Reply by jamessimone** [reply]
@melissajhansen given that this was also associated with #232, I am wondering if `v1.3.24` addresses this issue for you. Otherwise, something that would help me dig in a little further is if you update the `Logging Debug Level` Rollup Plugin Parameter CMDT record from `DEBUG` to `FINER` or `FINEST` - some of the evaluation that takes place specific to calc item where clauses I only log out on those more granular log settings to avoid slowing things down in normal operations. Looking forward to your response!
